neuvue_project/workspace/views/ng_state_plugins.py

```diff
@@ -15,24 +16,23 @@ def post(self, request, *args, **kwargs):
 
         # Make sure input state is well formed
         try:
+            namespace = data["namespace"]
             ng_state = dict(data["ng_state"])
         except:
             return HttpResponse("Ng state could not be parsed into dict", status=500)
 
         # Make sure plugin exists
         try:
-            ng_state_plugin = data["ng_state_plugin"]
-            ng_state_function = NG_STATE_PLUGINS[ng_state_plugin]
+            # Get name and params from namespace
+            namespace = Namespace.objects.get(namespace=namespace)
+            ng_state_plugin_name = namespace.ng_state_plugin.name
+            ng_state_plugin_params = namespace.get_effective_plugin_params()
+            print(type(ng_state_plugin_params))
+            print(ng_state_plugin_params)
```

Comment:
Should we remove these print statements?

Reply:
good catch!